src/nuclio/common/components/edit-item/edit-item.component.js

```diff
@@ -797,6 +794,19 @@
             }
         }
 
+        /**
+         * Removes volume classes except `selectedClass`
+         * @param {string} selectedClass
+         */
+        function cleanVolumeClasses(selectedClass) {
```

Comment:
Very nice!
I'd rename it to `cleanOtherVolumeClasses`

Reply:
Done!